### PR TITLE
Fix issue errormessages

### PR DESCRIPTION
**Pull request by ashishpriyadarshiCIC**
Change in the url used in google/daily.jl as  the current one is not working and change in the error message to make it more explanatory

### CODE REVIEW

File: src/google/daily.jl [hunk 1]
```diff
@@ -3,7 +3,7 @@ struct DataReaderResponseGoogleDaily <: AbstractDataReaderResponse
 end
 
 function get(dr::DataReaderGoogleDaily, symb::DataSymbol, dt_start::DateTime, dt_end::DateTime)
-    url = "http://www.google.com/finance/historical"
+    url = "http://finance.google.com/finance/historical?"
```
Comment (scls19fr) [inline src/google/daily.jl]: Is "?" really requires here?

File: src/DataReaders.jl [hunk 2]
```diff
@@ -112,7 +112,7 @@ module DataReaders
         #println(r.response)
         #if r.response.status / 100 != 2
         if r.status / 100 != 2
-            error("Error downloading data")
+            error("""Error : Not able to process request on the URl '""",url,"""' """)
```
Comment (scls19fr) [inline src/DataReaders.jl]: It should be URL, not URl

Only one " is nessary to begin string delimitation
Triple quotes are used if you want to add one " in a string without escaping char (using backslash)

you might use $(url) or simply $url for string interpolation
See https://en.m.wikibooks.org/wiki/Introducing_Julia/Strings_and_characters 
string interpolation

Comment (ashishpriyadarshiCIC) [reply src/DataReaders.jl]: Thank you Sir for showing me my mistakes, I will correct it right now.

File: src/google/daily.jl (hunk 1 above)
Comment (ashishpriyadarshiCIC) [reply src/google/daily.jl]: No Sir, it is not required. I'm removing it from the code

File: src/DataReaders.jl [hunk 3]
```diff
@@ -112,7 +112,7 @@ module DataReaders
         #println(r.response)
         #if r.response.status / 100 != 2
         if r.status / 100 != 2
-            error("Error downloading data")
+            error("Not able to process request on the URL: ",url)
```
Comment (scls19fr) [inline src/DataReaders.jl]: Could you try something like

    error("Error $(r.status) - not able to process request on the URL: $url with $query")

**Comment by scls19fr** [comment]
Closes #32 
Closes #28 

Thanks @ashishpriyadarshiCIC 

Next step it to tackle docstring / doc issue.

**Pull request merged**